app/src/main/res/layout/activity_login.xml

```diff
@@ -2,92 +2,107 @@
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:background="@color/white"
     android:layout_height="match_parent"
     android:layout_width="match_parent"
     android:orientation="vertical">
 
+    <include
+        layout="@layout/toolbar"
+        android:visibility="gone" />
+
     <androidx.core.widget.NestedScrollView
-        android:fitsSystemWindows="true"
+        android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_width="match_parent">
+        android:orientation="vertical">
 
         <LinearLayout
             android:id="@+id/ll_login"
-            android:layout_height="wrap_content"
             android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="@dimen/layout_padding_150dp"
             android:orientation="vertical"
-            android:paddingEnd="@dimen/login_padding_right"
-            android:paddingLeft="@dimen/login_padding_left"
-            android:paddingRight="@dimen/login_padding_right"
-            android:paddingStart="@dimen/login_padding_left"
-            android:paddingTop="@dimen/login_padding_top">
-
-            <ImageView
-                android:layout_gravity="center_horizontal"
-                android:layout_height="@dimen/login_height"
-                android:layout_marginBottom="@dimen/login_margin_bottom"
-                android:layout_width="wrap_content"
-                android:src="@drawable/mifos_logo"/>
-
-            <!-- Username  Label -->
-            <com.google.android.material.textfield.TextInputLayout
-                android:id="@+id/til_username"
+            android:padding="@dimen/layout_padding_30dp">
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/productsans"
```

Comment:
why we are introducing a new font?

Reply:
this was the font that resembled the most to the one given in the **redesign ppt** on the Github page.

Reply:
is this font used in other projects also, where redesigning is done?

Reply:
yes. I have used this font in my home screen redesign pr #1626